tests/basic/provider.tf

```diff
@@ -0,0 +1,6 @@
+provider "azurerm" {
+  skip_provider_registration = "true"
+
+  features {
+  }
+}
```

Comment:
no tests provider described,
make sure `terraform test` works

Reply:
Azure provider is described like this and it's enough for running the code, have tested, it works